lab5.py

```diff
@@ -0,0 +1,62 @@
+
+class Sneakers:
+    def __init__(self, brand, size, color, price, quantity, material, numberOfSales):
+        self.brand = brand
+        self.size = size
+        self.color = color
+        self.price = price
+        self.quantity = quantity
+        self.material = material
+        self.numberOfSales = numberOfSales
+
+    def __str__(self):
+        return f"{self.brand} ({self.color}, Size {self.size}): ${self.price}, Available: {self.quantity}"
+
+class SportShoesStore:
+    def __init__(self):
+        self.inventory = []
+
+    def add_sneakers(self, sneakers):
+        self.inventory.append(sneakers)
+
+    def sort_by_price(self):
+        self.inventory.sort(key=lambda x: x.price)
+
+    def sort_by_quantity(self):
+        self.inventory.sort(key=lambda x: x.quantity, reverse=True)
+
+    def top_sellers(self, n):
```

Comment:
Change argument name "n" to some other name, for example name.

Reply:
fixed